examples/pubsub_scheduled/README.md

```diff
@@ -2,15 +2,18 @@
 
 This example module schedules a job to publish a message to a Pub/Sub topic every 5 minutes, which will trigger a CloudFunctions function.
 
+Note that this example requires an app_engine app in the specified project/region. This is because scheduled functions are dependent on the project being configured with app engine.
```

Comment:
We don't need to repeat this information; just mention that the example does not handle the dependency and link back to the section in the root module README for more information.